core/src/main/java/com/linecorp/armeria/server/encoding/HttpEncoders.java

```diff
@@ -76,6 +78,12 @@ static OutputStream getEncodingOutputStream(HttpEncodingType encodingType, Outpu
                     throw new IllegalStateException(
                             "Error writing brotli header. This should not happen with byte arrays.", e);
                 }
+            case SNAPPY:
```

Comment:
Could you add tests for the new compression algorithm to `HttpEncodersTest`, `DecodingServiceTest`, `DecodingClientTest`, and so on?